src/src/Entity/PortfolioItem.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\PortfolioItemRepository;
+use Doctrine\DBAL\Types\Types;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(repositoryClass: PortfolioItemRepository::class)]
+class PortfolioItem
+{
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column]
+    private ?int $id = null;
+
+    #[ORM\Column(length: 255, nullable: true)]
+    private ?string $title = null;
+
+    #[ORM\Column(nullable: true)]
+    private ?array $FirstImage = null;
```

Comment:
Property names in PHP should start with a lowercase letter (i.e. `$firstImage`).